src/control/cmd/dmg/system.go

```diff
@@ -177,7 +177,7 @@ func (cmd *systemEraseCmd) Execute(_ []string) error {
 // systemStopCmd is the struct representing the command to shutdown DAOS system.
 type systemStopCmd struct {
 	baseRankListCmd
-	Force bool `long:"force" description:"Force stop DAOS system members"`
+	Force bool `long:"force" description:"Currently ignored"`
```

Comment:
IMO this changed line isn't helpful. It will have to be changed back, and it doesn't really tell the admin anything useful. "Oh, it's ignored? So then force stop doesn't work? Well then, how do I forcibly stop the system?" 

You can see how this change may have the opposite effect to what you intended... I think the description should be the same and the flag should just be a no-op so that everyone doesn't have to change their scripts.